test/Main.purs

```diff
@@ -110,3 +110,42 @@ main = do
             true
       _ ->
         false
+
+  assertParse "Records with raw string labels"
+    ("""
+    module Foo where
```

Comment:
You shouldn’t need to parse an entire module. The test function can parse isolated expressions.